<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="EditableSceneWidget" name="sceneWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QToolButton" name="moveTool">
          <property name="text">
           <string>M</string>
          </property>
          <property name="icon">
           <iconset resource="tool-icons.qrc">
            <normaloff>:/icons/icon-move.png</normaloff>:/icons/icon-move.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="circleTool">
          <property name="text">
           <string>C</string>
          </property>
          <property name="icon">
           <iconset resource="tool-icons.qrc">
            <normaloff>:/icons/icon-circle.png</normaloff>:/icons/icon-circle.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="triangleTool">
          <property name="text">
           <string>T</string>
          </property>
          <property name="icon">
           <iconset resource="tool-icons.qrc">
            <normaloff>:/icons/icon-triangle.png</normaloff>:/icons/icon-triangle.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="rectangleTool">
          <property name="text">
           <string>R</string>
          </property>
          <property name="icon">
           <iconset resource="tool-icons.qrc">
            <normaloff>:/icons/icon-rect.png</normaloff>:/icons/icon-rect.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="nodeSizeSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>EditableSceneWidget</class>
   <extends>QWidget</extends>
   <header>../common/scene/editablescenewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="tool-icons.qrc"/>
 </resources>
 <connections/>
</ui>
